ansible_collections/arista/avd/roles/eos_designs/templates/base/daemon-terminattr.j2

```diff
@@ -8,8 +8,8 @@
 {%     if cvp_instance_ip is arista.avd.defined %}
 {%         if "arista.io" not in cvp_instance_ip %}
 {%             do cvp_on_prem.ips.append(cvp_instance_ip) %}
-{%         elif "arista.io" in cvp_ip %}
-{%             do cvaas.ips.append(cvp_ip) %}
+{%         elif "arista.io" in cvp_instance_ip %}
```

Comment:
As we already checked in line 9, if "arista.io" exists in cvp_instance_ip or not. Simply an ```else``` should work without any issue.

IMO, we can modify the line 9-12 to (Prefer to avoid not(logic) )

```
{%     if cvp_instance_ip is arista.avd.defined %}
{%         if "arista.io" in cvp_instance_ip %}
{%             do cvaas.ips.append(cvp_instance_ip) %}
{%         else %}
{%             do cvp_on_prem.ips.append(cvp_instance_ip) %}
```

Reply:
If it's ok for everyone, we can modify the below logic as well.

Reply:
If we change that, I would swap to `if "arista.io" in cvp_instance_ip` and `else` for the onprem.

Reply:
Implemented the suggestion, thanks!

Reply:
Thanks. I took the liberty of also fixing auth for cvaas. @noredistribution can you take a look at the last commit here and see if it looks right?